### DISCUSSION TITLE
頂大廣開AI通識課 修課劇增／相關課程成長2~5成 不分科系學生都有機會修習

### DISCUSSION THREAD

**Discussion by Patsy17** [1101-GS4719-C/Python-Community-GS4719-C#1804] [General]
AI人工智慧、資通訊科技浪潮興起，也為大學通識課內容帶來質變，多所頂尖大學近年陸續開設相關課程，成長幅度二成至五成，修課學生比率更是倍數成長，產生「通識課程AI化」現象，非本科學生也可增加AI知能。

資料來源：https://news.ltn.com.tw/news/life/paper/1616509

因應人工智慧的趨勢，我認為不管是什麼科系的學生都可以多去選修這些資訊科技的課程，
來增加自己的競爭力，也可以更自己有更多元的職缺來選擇！
而且學校的資源很多，更應該要好好把握這些資源～
人才很多，如果不一直持續精進自我，那就會被許多人才比下去

**Comment by eric202501** [comment]
人工智慧和資訊科技的發展對大學通識課程產生了深遠的影響，跨科系選修這些資訊科技的課程確實有助於擴展自己的知識面，提高競爭力。現今社會，之後不論在哪個領域，都應該需要具備基本的資訊科技能力
